common/lib/xmodule/xmodule/templates/html/image_modal.yaml

```diff
@@ -1,7 +1,15 @@
 ---
 metadata:
-    display_name: Full Screen Image
+    display_name: Full Screen Image Tool
 data: |
-        <h2>Full Screen Image</h2>
-        <p>Use the full screen image tool to allow students to open the image in the full screen, zoom in on the image, and drag the image within the screen.</p>
+        <h2>Full Screen Image Tool</h2>
+        <p>Use the Full Screen Image tool to allow learners to open the image in the full screen, zoom in on the image, and drag the image within the screen.</p>
+        <p>With the Full Screen Image tool, learners can see the image's detail as well as its context within the unit.</p>
+        <p>Modify this template as needed.</p>
+        <ol>
+        <li>Replace the value of the link's <strong>href</strong> attribute with the path to the full size image. Do not change the value of the class attribute.</li>
+        <li>Replace the value of the image's <strong>src</strong> attribute with the path to the image that will appear in the unit before a learner opens in full screen mode.</li>
+        </ol>
+        <p><strong>Note</strong>: Test viewing the image in the full screen in the LMS; you cannot view it in full screen from within Studio.</p>
+
```

Comment:
It should be noted that alt text for a linked image should describe the *action* or *destination* of the link.  In this instance, it's also important to describe the image.  So the guidance should be to do BOTH describe the image and the action of clicking on it i.e. ```alt="View in full screen: Photo of The Stanford Hills"```.  This is required because a screen reader user would imply that clicking on the image would take them to more information about The Stanford Hills, not to a full screen image of them.